src/main/java/lotto/domain/WinningNumbers.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import lotto.exception.LottoException;
+import lotto.exception.LottoExceptionType;
+
+public class WinningNumbers {
+
+    private final Lotto winner;
+    private final LottoNumber bonusNumber;
+
+    public WinningNumbers(final Lotto winner, final LottoNumber bonusNumber) {
+        validate(winner, bonusNumber);
+        this.winner = winner;
+        this.bonusNumber =  bonusNumber;
+    }
+
+    private void validate(final Lotto winner, final LottoNumber bonusNumber) {
+        if (winner.getLottoNumber().contains(bonusNumber)) {
+            throw new LottoException(LottoExceptionType.DUPLICATE_BONUS_NUMBER);
+        }
+    }
```

Comment:
Lotto 의 contains 메서드를 활용해 보면 좋을 것 같아요 😄 

```suggestion
    private void validate(final Lotto winner, final LottoNumber bonusNumber) {
        if (winner.contains(bonusNumber)) {
            throw new LottoException(LottoExceptionType.DUPLICATE_BONUS_NUMBER);
        }
    }
```

Reply:
수정하였습니다.